### PR TITLE
Clarify 'per user' project strategy in case if users didn't have enough privileges

### LINKED ISSUE #19572: [doc] Not able to configure "per user" project strategy because users didn't had enough privileges.
### Is your task related to a problem? Please describe.
To switch to "per user", we had to ask an admin to manually create the user namespace and grant the user's admin privileges. 

### Describe the solution you'd like
That is a requirement? if it's really a requirement, it is not documented. In any case, we should indicate what ServiceAccount is used to create the user's projects if they do not exist in https://www.eclipse.org/che/docs/che-7/installation-guide/configuring-namespace-strategies/#_one_namespace_per_user_strategy.

### Describe alternatives you've considered
n/a
### Additional context
https://issues.redhat.com/browse/CRW-1716

### PR DESCRIPTION
**Pull request by skabashnyuk**
### What does this PR do?
Clarify 'per user' project strategy in case if users didn't have enough privileges.

### What issues does this PR fix or reference?
Fixes https://github.com/eclipse/che/issues/19572

### Specify the version of the product this PR applies to.
current.

### PR Checklist

As the author of this Pull Request I made sure that:

- [ ] `vale` has been run successfully against the PR branch
- [ ] Link checker has been run successfully against the PR branch
- [ ] Documentation describes a scenario that is already covered by QE tests, otherwise an issue has been created and acknowledged by Che QE team
- [ ] Changed article references are updated where they are used (or a redirect has been set up on the docs side):
    - [ ] Dashboard [branding.json](https://github.com/eclipse/che-dashboard/blob/master/src/components/branding/branding.json)
    - [ ] Chectl [constants.ts](https://github.com/che-incubator/chectl/blob/master/src/constants.ts)

### CODE REVIEW

**Review by azatsarynnyy — APPROVED** [review]

File: modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc [hunk 1]
```diff
@@ -60,6 +60,11 @@ Be aware that the `_<userid>_` placeholder is evaluated into a 36 character long
 ====
 
 WARNING: For strategies where creating new {orch-namespace} is needed, make sure that `che` ServiceAccount has enough permissions to do so. With OpenShift OAuth, the authenticated User must have privileges to create new {orch-namespace}.
+`self-provisioner` cluster role should be linked to  `system:authenticated:oauth` group.
```
Comment (yhontyk) [inline modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc]:
```suggestion
the `self-provisioner` cluster role should be linked to the `system:authenticated:oauth` group.
```

File: modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc [hunk 2]
```diff
@@ -60,6 +60,11 @@ Be aware that the `_<userid>_` placeholder is evaluated into a 36 character long
...
+
+WARNING: In situations when  `che` ServiceAccount has not enough permissions creating new {orch-namespace} or with OpenShift OAuth
```
Comment (yhontyk) [inline modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc]: I suggest restructuring this sentence slightly, smth like:
``
WARNING: Yout must use <<pre-creating-namespace>> if `che` ServiceAccount does not have enought permissions when creating new {orch-namespace}, with OpenShift OAuth when a `self-provisioner` cluster role is not linked to the`system:authenticated:oauth` group, or in other situations when {prod-short} cannot create namespaces.``

**Review by yhontyk — APPROVED** [review]
Apologize for the delay @skabashnyuk 
Approving with a couple of suggestions.

File: modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc (hunk 2 above)
Comment (MichalMaler) [reply modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc]: Thx Yana, I will find a way how to put the info in the original context.

```
WARNING: Use <<pre-creating-namespace>> when:

* `che` ServiceAccount does not have enought permissions when creating new {orch-namespace}

 * OpenShift OAuth with a `self-provisioner` cluster role is not linked to the`system:authenticated:oauth` group
 
* {prod-short} cannot create namespaces
```

Comment (skabashnyuk) [reply modules/installation-guide/partials/proc_configuring-namespace-strategies.adoc]: done

**Review by MichalMaler — APPROVED** [review]

**Pull request merged**